backend/lib/database/postgres/postgis.py

```diff
@@ -46,18 +49,17 @@ def install():
     """Install Postgis to Postgres on AWS."""
     # In the postgres database.
```

Comment:
What does this comment mean? Does this script need to be run on the RDS instance?